pyvrp/cpp/search/SwapTails.h

```diff
@@ -0,0 +1,29 @@
+#ifndef PYVRP_SEARCH_SWAPTAILS_H
+#define PYVRP_SEARCH_SWAPTAILS_H
+
+#include "LocalSearchOperator.h"
+
+namespace pyvrp::search
+{
+/**
+ * SwapTails(data: ProblemData)
+ *
+ * Given two nodes :math:`U` and :math:`V`, tests whether replacing the arc of
+ * :math:`U` to its successor :math:`n(U)` and :math:`V` to :math:`n(V)` by
+ * :math:`U \rightarrow n(V)` and :math:`V \rightarrow n(U)` is an improving
+ * move.
+ */
```

Comment:
Can we add a note here similar that this is known as `2-OPT*`?

Reply:
Sure thing!